timesketch/frontend-ng/src/components/LeftPanel/SigmaRule.vue

```diff
@@ -16,39 +16,87 @@ limitations under the License.
 <template>
   <div>
     <v-row no-gutters class="pa-3" :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'">
-      <div @click="expanded = !expanded" style="cursor: pointer; font-size: 0.9em">
+      <div @click="expanded = !expanded" class="SigmaRuleTitle">
         <v-icon v-if="!expanded">mdi-chevron-right</v-icon>
         <v-icon v-else>mdi-chevron-down</v-icon>
         {{ sigmaRule.title }}
+        <v-chip rounded x-small class="mr-2">{{ sigmaRule.status }}</v-chip>
       </div>
       <v-spacer></v-spacer>
-      <div><v-icon>mdi-dots-vertical</v-icon></div>
+      <div>
+        <v-menu offset-y>
+          <template v-slot:activator="{ on, attrs }">
+            <v-avatar>
+              <v-btn small icon v-bind="attrs" v-on="on">
+                <v-icon>mdi-dots-vertical</v-icon>
+              </v-btn>
+            </v-avatar>
+          </template>
+          <v-card>
+            <v-list>
+              <v-list-item-group color="primary">
+                <v-list-item>
+                  <v-list-item-icon>
+                    <v-icon>mdi-brightness-6</v-icon>
```

Comment:
Please change to the correct icons and align them:

This is what is looks like now:
<img width="667" alt="Screen Shot 2023-01-13 at 13 26 38" src="https://user-images.githubusercontent.com/316362/212320361-66957b53-bd96-4ccf-8ec8-6abde46e994c.png">

Reply:
done